tests/test_pytest_logger.py

```diff
@@ -215,6 +217,33 @@ def test_file_handlers(testdir, conftest_py, test_case_py):
     ])
 
 
+@pytest.mark.skipif(sys.version_info < (3, 6, ) or win32,
+                    reason="Requires linux and python 3.6+")
```

Comment:
Why would python 3.6+ be necessary to pull the trick? I don't see why making symlinks would be impossible in python 3.5.

Reply:
there are some differences in os.path interface. Probably I can make it work also on 3.5 - but it will require some if-o-logy... 
> `TypeError: relpath() argument must be str or bytes, not 'LocalPath'"`
https://travis-ci.org/aurzenligl/pytest-logger/jobs/519233877

Reply:
I'd prefer it to work on as many platforms as possible. I haven't investigated this deeply, pytest might pick different type for path depending on python version. Maybe it's related to switch to pathlib in pytest?
https://github.com/hackebrot/pytest-cookies/issues/27

Reply:
fixed.